### ISSUE TITLE
Upgrading from 0.13.8 to 1.0.0

### ISSUE THREAD

**Issue by rwb196884** [davidhu2000/react-spinners#584]
In my code is

```
import { DotLoader } from 'react-spinners/';
```

and after upgrading it nolonger builds:
```
 error TS7016: Could not find a declaration file for module 'react-spinners'. 'C:/Work/Azure/TruthPortalApp/client/node_modules/react-spinners/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/react-spinners` if it exists or add a new declaration (.d.ts) file containing `declare module 'react-spinners';`
```

**Comment by rwb196884** [comment]
1.0.0 is some kind of fuck up and 0.13.8 _is_ the latest version?

**Comment by davidhu2000** [comment]
0.13.8 is the latest https://github.com/davidhu2000/react-spinners/issues/563#issuecomment-1461236830

**Comment by davidhu2000** [comment]
when i have time, i'll fix some existing issues and publish 1.0.1, so that should resolve this going forward
